lld/COFF/LTO.cpp

```diff
@@ -126,8 +127,23 @@ BitcodeCompiler::BitcodeCompiler(COFFLinkerContext &c) : ctx(c) {
         llvm::heavyweight_hardware_concurrency(ctx.config.thinLTOJobs));
   }
 
+  lto::LTO::LTOKind Kind = lto::LTO::LTOK_Default;
```

Comment:
To simplify this code, please put this switch into a static helper. It's similar in spirit to this guideline about predicate loops: https://llvm.org/docs/CodingStandards.html#turn-predicate-loops-into-predicate-functions

This way, you don't need the break or a local variable:
```
lto::LTO::LTOKind translateLTOKind(LTOKind k) {
  switch (k) {
  case LTOKind::Default:
    return lto::LTO::LTOK_Default;
...
```

Other nit is to prefer lower-cased variables in LLD. That was a style divergence that we never resolved.

Reply:
> That was a style divergence that we never resolved.

Can we not converge now with the rest of LLVM? I feel this was Rui’s preference and he’s not involved anymore with the project.

Reply:
I think our first priority should probably be to remain in alignment with the other LLD ports, and they use lower case variables. We'd need input from the other LLD contributors, particularly @maskray.